📚: add note convention mod (#22)
mc-datapacks/en: src/style_guideline/reloading_message.md

You **must not** send an automatic message when the world is reloaded. It will clutter the chat unnecessarily and create a bad experience for the users.

Most of the time, the message will be an installation message, in which case you can use [Datapack Advancement](../conventions/datapack_advancement.md) instead.

--------------------

> ### **Note from Convention Mod (Optional)**

Reviewer: Consider a different title

Author: ```suggestion
### **Additional Guideline (Optional)**
```

Reviewer: No.

> ### **Exception cases**
> 
> > **Case 1:** a custom advancement to keep track of who has seen the message.
> ```json
> {
>   "criteria": {
>     "trigger": {
>       "trigger": "minecraft:tick"
>     }
>   },
>   "rewards": {
>     "function": "..."
>   }
> }
> ```

Author: ```suggestion
### **Exception cases**

**Case 1:** Custom advancement 
- Simply use this a custom advancement to keep track of who has seen the message.

```json
{
  "criteria": {
    "trigger": {
      "trigger": "minecraft:tick"
    }
  },
  "rewards": {
    "function": "..."
  }
}
```
```

> **Case 2:** debugging reload messages that normal players shouldn't see.

``convention:datapack/setup``
```mcfunction
tellraw @a[name=Owner] <...>
tellraw @a[tag=convention.debug] <...>

> ```

Reviewer: Lack of text description; fit this inside a paragraph instead of putting it under a title header. Don't rely on code examples alone.

Author: ```suggestion
### **Exception cases**

**Case 1:** Custom advancement 
- Simply use this a custom advancement to keep track of who has seen the message.

```json
{
  "criteria": {
    "trigger": {
      "trigger": "minecraft:tick"
    }
  },
  "rewards": {
    "function": "..."
  }
}
```

**Case 2:** Debug message
- This example is debugging reload messages that normal players shouldn't see. 

```mcfunction
tellraw @a[name=Owner] <...>
tellraw @a[tag=convention.debug] <...>
```
```

Author: ```suggestion
### **Exception cases**

> **Case 1:** a custom advancement to keep track of who has seen the message.
```json
{
  "criteria": {
    "trigger": {
      "trigger": "minecraft:tick"
    }
  },
  "rewards": {
    "function": "..."
  }
}
```
> **Case 2:** debugging reload messages that normal players shouldn't see.

``convention:datapack/setup``
```mcfunction
tellraw @a[name=Owner] <...>
tellraw @a[tag=convention.debug] <...>
```
```

Author: 📚: fix suggested

Reviewer: This still isn't fixed.

> > **Case 2:** debugging reload messages that normal players shouldn't see.
> 
> ``convention:datapack/setup``
> ```mcfunction
> tellraw @a[name=Owner] <...>
> tellraw @a[tag=convention.debug] <...>
> ```

Author: ```suggestion

**Case 2:** Debug message
- This example is debugging reload messages that normal players shouldn't see. 

```mcfunction
tellraw @a[name=Owner] <...>
tellraw @a[tag=convention.debug] <...>
```
```
